sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Microsoft.Azure.WebJobs.Extensions.EventHubs.csproj

```diff
@@ -11,11 +11,13 @@
   <ItemGroup>
     <PackageReference Include="Azure.Storage.Blobs" />
     <PackageReference Include="Microsoft.Azure.WebJobs" />
+    
+    <!-- TODO: Switch to centralized versions when Microsoft.Extensions.Azure GA -->
+    <PackageReference Include="Microsoft.Extensions.Azure" VersionOverride="1.1.0-beta.1" />
```

Comment:
This is necessary to get the nightly feed to build?

Reply:
Package publishing happens for a particular service directory and Microsoft.Extensions.Azure lives in another one. It can result in a broken dependency if we keep the project ref across service directories.